dependencycheck.properties

```diff
@@ -1,13 +1,19 @@
 category=Integration
 description=Integrates Dependency-Check reports into SonarQube
 homepageUrl=https://github.com/dependency-check/dependency-check-sonar-plugin
-archivedVersions=2.0.7,2.0.8,3.0.0,3.0.1
-publicVersions=3.1.0
+archivedVersions=2.0.7,2.0.8,3.0.0,3.0.1,3.1.0
+publicVersions=4.0.0
 defaults.mavenGroupId=org.sonarsource.owasp
 defaults.mavenArtifactId=sonar-dependency-check-plugin
 
+4.0.0.description=Support SonarQube 10
+4.0.0.sqVersions=[9.9.1, LATEST]
+4.0.0.date=2023-05-16
+4.0.0.changelogUrl=https://github.com/dependency-check/dependency-check-sonar-plugin/releases/tag/4.0.0
+4.0.0.downloadUrl=https://github.com/dependency-check/dependency-check-sonar-plugin/releases/download/4.0.0/sonar-dependency-check-plugin-4.0.0.jar
+
 3.1.0.description=Support dependency-check 8.0.0
-3.1.0.sqVersions=[8.9.7, LATEST]
+3.1.0.sqVersions=[8.9.7, 9.9]
```

Comment:
The range end needs to be set to the current version. Otherwise your users on 10.0 won't be prompted to upgrade.

Reply:
Okay. Then this should end with `9.9.*` or `9.9.1` explicitly since there is a point version out.